charts/pwa/README.md

```diff
@@ -177,47 +177,33 @@ The value for `stop` determines the duration in seconds after the job is forcefu
 
 ## Split Ingress
 
-Sometimes customers only want to go live with a subset of their domains, but want to keep the rest hidden behind IP whitelisting. Therefore, a second Ingress object was implemented to address this use case.
-`ingresssplit` is disabled by default. To implement it in your project, follow the example below:
+Sometimes customers only want to go live with a subset of their domains, but want to keep the rest hidden behind IP whitelisting. Therefore, a second host object can be added to the ingress config to address this use case.
+To implement it in your project, follow the example below:
 
 ```yaml
 # This Ingress has IP whitelisting, so it is hidden from the world, except for IPs xxx.xxx.xxx.xxx and yyy.yyy.yyy.yyy
 ingress:
   enabled: true
   className: nginx
-  annotations:
-    kubernetes.io/tls-acme: "false"
-    # xxx.xxx.xxx.xxx and yyy.yyy.yyy.yyy are valid IP-Addresses to be whitelisted
-    configuration-snippet: |-
-      satisfy any;
-      allow xxx.xxx.xxx.xxx;
-      allow yyy.yyy.yyy.yyy;
-      deny all;
   hosts:
   # in case multiple PWA instances will be deployed into the given environment namespace, a postfix
   # has to be added to the hostname: i.e. ${pwa-hostname}-edit
   - host: ${pwa_hostname}.pwa.intershop.de
-    paths:
-    - path: /
-      pathType: ImplementationSpecific
-  tls:
-  - secretName: tls-star-pwa-intershop-de
-# This is the 2nd ingress that is "live" and visible from everywhere
-ingresssplit:
-  enabled: true
-  className: nginx
-  annotations:
-    kubernetes.io/tls-acme: "false"
-    nginx.ingress.kubernetes.io/
-  hosts:
-  # in case multiple PWA instances will be deployed into the given environment namespace, a postfix
-  # has to be added to the hostname: i.e. ${pwa-hostname}-split-edit
+    secretName: tls-star-pwa-intershop-de
+    annotations:
```

Comment:
You would have this same snippet of code (annotation definition, which holds the IP Whitelisting) in your values.yaml for every non-live host. This is not manageable and quite prone to errors/not easy to see through a 5000 lines values.yaml. Also if you have to whitelist another IP, you would need to do it multiple times, instead of only once for the non-live ingress. Also you would have like 300 ingresses on the Kubernetes Cluster, instead of simply having 2.